### PR TITLE
RA-1773: Corrected inconsistencies with 'require' label

### PR DESCRIPTION
**Pull request by druchniewicz**
Related ticket: https://issues.openmrs.org/browse/RA-1773

Related PR with changes in registrationapp module - https://github.com/openmrs/openmrs-module-registrationapp/pull/78

### CODE REVIEW

File: omod/src/main/webapp/fragments/field/multipleInputDate.gsp [hunk 1]
```diff
@@ -101,7 +101,7 @@
 
 <p id="${config.id}">
     <label for="${ config.id }-field">
-        ${ config.label } <% if (config.classes && config.classes.contains("required")) { %><span>(${ ui.message("emr.formValidation.messages.requiredField.label") })</span><% } %>
+        ${ config.label } <% if (config.classes && config.classes.contains("requiredTitle")) { %><span>(${ ui.message("emr.formValidation.messages.requiredField.label") })</span><% } %>
```
Comment (mogoodrich) [inline omod/src/main/webapp/fragments/field/multipleInputDate.gsp:104]: Why this change from required to requiredTitle?

Comment (druchniewicz) [reply omod/src/main/webapp/fragments/field/multipleInputDate.gsp:104]: I responded in related PR - https://github.com/openmrs/openmrs-module-registrationapp/pull/78/
